app/src/main/java/com/sdp13epfl2021/projmag/database/UserDataFirebase.kt

```diff
@@ -108,4 +109,19 @@ class UserDataFirebase(
             )
         }
     }
+
+    override fun pushCv(
+        cv: CurriculumVitae,
+        onSuccess: () -> Unit,
+        onFailure: (java.lang.Exception) -> Unit
```

Comment:
Exception is a little more readable than java.lang.Exception
```suggestion
        onFailure: (Exception) -> Unit
```